R/print.data.table.R

```diff
@@ -118,9 +119,9 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
     }
     rownames(toprint) = format(rownames(toprint), justify="right")
     if (col.names == "none") {
-      cut_colnames(print(toprint, right=TRUE, quote=quote))
+      cut_colnames(print(toprint, right=TRUE, quote=quote, na.print=na.print))
```

Comment:
we really shouldn't need all this repetition. but that's not relevant to this PR.

Reply:
#6089